src/main/resources/assets/resourcefulbees/lang/zh_cn.json

```diff
@@ -0,0 +1,171 @@
+{
+	"itemGroup.resourcefulbees" : "缤纷蜜蜂",
+
+	"resourcefulbees.information.bee_type.vanilla": "蜜蜂类型：原版",
+	"resourcefulbees.information.bee_type.unknown": "蜜蜂类型：未知",
+	"resourcefulbees.information.bee_type.custom": "蜜蜂类型：",
+	"resourcefulbees.shift_info": "按 SHIFT 查看信息",
+	"resourcefulbees.ctrl_info": "按 CTRL 查看多方块信息",
+
+	"honeycomb.resourcefulbees": "蜜脾",
+	"honeycomb_block.resourcefulbees": "蜜脾块",
+
+	"gui.resourcefulbees.jei.category.hive" : "蜂箱",
+	"gui.resourcefulbees.jei.category.centrifuge": "离心机",
+	"gui.resourcefulbees.jei.category.breeding" : "蜜蜂饲养",
+	"gui.resourcefulbees.category.centrifuge": "离心机",
+	"gui.resourcefulbees.jei.category.apiary": "养蜂场",
+	"gui.resourcefulbees.jei.category.fluid_to_fluid_mutation": "液液突变",
+	"gui.resourcefulbees.jei.category.block_to_fluid_mutation": "固液突变",
+	"gui.resourcefulbees.jei.category.fluid_to_block_mutation": "液固突变",
+	"gui.resourcefulbees.jei.category.block_to_block_mutation": "固固突变",
+	"gui.resourcefulbees.jei.category.mutation.info": "蜜蜂固体突变需要有效的蜂箱。",
+	"gui.resourcefulbees.centrifuge": "动力离心机",
+	"gui.resourcefulbees.beehive.smoked": "烟熏：",
+	"gui.resourcefulbees.beehive.honeylevel": "蜂蜜等级：",
+	"gui.resourcefulbees.beehive.num_combs": "蜜脾",
+	"gui.resourcefulbees.apiary.button.validate": "验证",
+	"gui.resourcefulbees.apiary.button.build": "建造",
+	"gui.resourcefulbees.apiary.button.build.creative": "仅限创造模式！",
+	"gui.resourcefulbees.apiary.button.preview.disable": "禁用预览",
+	"gui.resourcefulbees.apiary.button.preview.enable": "启用预览",
+	"gui.resourcefulbees.apiary.button.import" : "导入",
+	"gui.resourcefulbees.apiary.button.export" : "导出",
+	"gui.resourcefulbees.apiary.import.true" : "蜜蜂导入成功！",
+	"gui.resourcefulbees.apiary.import.false" : "蜜蜂导入失败！",
+	"gui.resourcefulbees.apiary.export.true" : "蜜蜂导出成功！",
+	"gui.resourcefulbees.apiary.export.false" : "蜜蜂导出失败！",
+	"gui.resourcefulbees.apiary.validated.true" : "养蜂场验证成功！",
+	"gui.resourcefulbees.apiary.validated.false" : "养蜂场验证失败！",
+	"gui.resourcefulbees.apiary.bee.ticks_in_hive": 蜂箱刻数",
```

Comment:
Can you please add the double quote here so that it wont error out and then ill merge it :)